src/cmd/setup-tachyon.js

```diff
@@ -477,6 +470,10 @@ Welcome to the Particle Tachyon setup! This interactive command:
 			config.productId = productId;
 		}
 
+		if (region) {
+			config.region = region;
+		}
+
```

Comment:
Region doesn't need to go into the config. It's only needed for the download.

Reply:
Done here! https://github.com/particle-iot/particle-cli/pull/794/commits/ba42e3e8991986611498c9650006493f15e7b38c